packages/cli-kit/src/path.ts

```diff
@@ -3,6 +3,8 @@ import commondir from 'commondir'
 import {relative, dirname, join, normalize, resolve, basename, extname, isAbsolute, parse} from 'pathe'
 import {findUp as internalFindUp} from 'find-up'
 import {fileURLToPath} from 'url'
+// eslint-disable-next-line node/prefer-global/url
+import type {URL} from 'url'
```

Comment:
Why not just delete this line?

Reply:
Because the older version of `@types/node` didn't expose `URL` type globally and therefore you are required to import it.